starport/templates/module/create/oracle/x/{{moduleName}}/keeper/grpc_query_oracle.go.plush

```diff
@@ -0,0 +1,23 @@
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"<%= modulePath %>/x/<%= moduleName %>/types"
+)
+
+func (k Keeper) OracleResult(c context.Context, req *types.QueryOracleRequest) (*types.QueryOracleResponse, error) {
```

Comment:
We must keep in mind that developers that want to develop a module with Bandchain may not know how does this solution work initially. We should add comments for all the methods we scaffold